modules/nixos/common/auto-upgrade.nix

```diff
@@ -2,7 +2,7 @@
 {
   system.autoUpgrade.enable = lib.mkDefault true;
   system.autoUpgrade.flake = "github:nix-community/infra";
-  system.autoUpgrade.dates = "hourly";
+  system.autoUpgrade.dates = "0/2:00";
```

Comment:
I don't want to disable autoupgrade yet as I think some of our services may sort of depend on it, two hours should give us a window to see something has gone down while not being down too long that it annoys people.